CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
 # Changelog
 
-## v3.0.0 31th May 2024
+## v3.0.0 14th Jun 2024
```

Comment:
Why 14th?

Reply:
It's just an estimate about the actual release date